src/screens/CurrencySelectModal.js

```diff
@@ -60,12 +67,38 @@ const getHasShownWarning = () =>
 const setHasShownWarning = () =>
   storage.set(STORAGE_IDS.SHOWN_SWAP_RESET_WARNING, true);
 
+const TINY_ANDROID_PADDING = 4;
+
 const TabTransitionAnimation = styled(Animated.View)(
   position.sizeAsObject('100%')
 );
 
+const Flex = styled.View({
+  flex: 1,
+});
+
+// This component reacts to the keyboard height and pushes the content up
+// It kinda mimcs the way adjustResize works but we won't break other screens with that
+// since when you change adjustResize during the navigation - it can push content
+// on the other screen up too
+function AndroidWrapper({ children }) {
+  const insets = useSafeArea();
+  const keyboardHeight = useKeyboardArea();
+
+  const marginBottom = keyboardHeight > 0 
+    // tiny padding adds a tiny space between the keyboard and the view
+    //  to match how it looks on iOS
+    ? insets.top + TINY_ANDROID_PADDING + keyboardHeight 
+    : 0;
+  return (
+    <Flex>
+      <Flex style={{ marginBottom }}>{children}</Flex>
+    </View>
```

Comment:
this is crashing the app, you meant to use `View` as the opening tag as well? is `Flex` right now

Reply:
i replaced View as open tag and i'm seeing this https://www.loom.com/share/f488df4bd57d4b21988ccd24ded6b20e not sure if that's what you had here before

Reply:
Yes, typo, gonna fix that